app/assets/stylesheets/application.css.scss

```diff
@@ -190,28 +190,31 @@ a.btn.btn-default:hover {
     font-size: 12px;
   }
 
-  .picture_background {
-    margin: -30px 5px 5px 5px;
-  }
-
   .photo-text {
     width: 100%;
     margin-left: -12px;
   }
 
+  .picture_background {
+    margin-top: -30px;
+    margin-left: 5px;
+    margin-right: 5px;
+    margin-bottom: 5px;
+  }
+
   .scholar {
+    background-color: #808080;
```

Comment:
Color literals like `#808080` should only be used in variable declarations; they should be referred to via variable everywhere else.